contracts/contract/network/RocketNetworkBalances.sol

```diff
@@ -33,22 +33,31 @@ contract RocketNetworkBalances is RocketBase, RocketNetworkBalancesInterface {
         setUint(keccak256("network.balances.updated.block"), _value);
     }
 
-    // The current RP network total ETH balance
+    // The current RP network total user ETH balance
     function getTotalETHBalance() override public view returns (uint256) {
         return getUint(keccak256("network.balance.total"));
     }
     function setTotalETHBalance(uint256 _value) private {
         setUint(keccak256("network.balance.total"), _value);
     }
 
-    // The current RP network staking ETH balance
+    // The current RP network staking ETH balance from users
     function getStakingETHBalance() override public view returns (uint256) {
         return getUint(keccak256("network.balance.staking"));
     }
     function setStakingETHBalance(uint256 _value) private {
         setUint(keccak256("network.balance.staking"), _value);
     }
 
+    // The current RP network total staking ETH balance from the minipool queue
+    //   - Note: this does not contribute to getTotalETHBalance, as it isn't from User deposits
+    function getStakingQueueETHBalance() override public view returns (uint256) {
+        return getUint(keccak256("network.balance.queuestaking"));
+    }
+    function setStakingQueueETHBalance(uint256 _value) private {
+        setUint(keccak256("network.balance.queuestaking"), _value);
+    }
```

Comment:
oDAO could calculate this as:
- a = `sum of nodeDepositBalance across efficient queue` (equivalently, sum of nodeDepositBalance where nodeDepositAssigned==False)
- b= ` number of minipools in efficient queue` (accounts for the 1 scrub ETH)
- c = `total user-deposited ETH since last assignment`
- `stakingQueueETHBalance = a - b - c`